src/50_Samples_%26_Templates/Product.html

```diff
@@ -269,62 +340,208 @@ <h4 class="category"><a href="/samples_templates/product_listing/preview/">More<
-     <ul class="hide-on-mobile">
-       <li on="tap:product-carousel.goToSlide(index=0)" tabindex="0" role="button" option="green" selected>
-         <amp-img src="/img/product1_60x40.jpg" width="60" height="40"></amp-img>
+    See below for more informations on how we use`amp-bind`. -->
+       <div class="product-gallery">
+    <ul class="show" [class]="selectedColor == 'green' ? 'show' : 'hide'">
+      <li tabindex="0" role="button">
+        <amp-img on="tap:AMP.setState(selectedSlide=0)"
+                  src="/img/green_apple_1_60x40.jpg"
+                  width="60" height="40"
+                  class="selected"
+                  [class]="selectedSlide == 0 ? 'selected' : '' ">
+        </amp-img>
+      </li>
+      <li tabindex="1" role="button">
+        <amp-img on="tap:AMP.setState(selectedSlide=1, selectedColor=green)"
+                  src="/img/green_apple_2_60x40.jpg"
+                  width="60" height="40"
+                  [class]="selectedSlide == 1 ? 'selected' : '' ">
+        </amp-img>
+      </li>
+    </ul>
+
+     <ul class="hide" [class]="selectedColor == 'golden' ? 'show' : 'hide'">
+       <li tabindex="0" role="button">
+         <amp-img on="tap:AMP.setState(selectedSlide=0)"
+                  src="/img/product1_alt1_60x40.jpg"
+                  width="60" height="40"
+                  [class]="selectedSlide == 0 ? 'selected' : '' ">
+        </amp-img>
        </li>
-       <li on="tap:product-carousel.goToSlide(index=1)" tabindex="1" role="button" option="golden">
-         <amp-img src="/img/product1_alt1_60x40.jpg" width="60" height="40"></amp-img>
+     </ul>
+
+     <ul class="hide" [class]="selectedColor == 'red' ? 'show' : 'hide'">
+       <li tabindex="0" role="button">
+         <amp-img on="tap:AMP.setState(selectedSlide=0)"
+                  src="/img/red_apple_1_60x40.jpg"
+                  width="60" height="40"
+                  [class]="selectedSlide == 0 ? 'selected' : '' ">
+         </amp-img>
        </li>
-       <li on="tap:product-carousel.goToSlide(index=2)" tabindex="2" role="button" option="red">
-         <amp-img src="/img/product1_alt2_60x40.jpg" width="60" height="40"></amp-img>
+       <li tabindex="1" role="button">
+         <amp-img on="tap:AMP.setState(selectedSlide=1, selectedColor=red)"
+                  src="/img/red_apple_2_60x46.jpg"
+                  width="60" height="40"
+                  [class]="selectedSlide == 1 ? 'selected' : '' ">
+         </amp-img>
        </li>
      </ul>
-    <amp-carousel id="product-carousel" width="1024" height="682" layout="responsive" type="slides">
-        <amp-img src="/img/product1_1024x682.jpg" width="1024" height="682" layout="responsive"></amp-img>
-        <amp-img src="/img/product1_alt1_1024x682.jpg" width="1024" height="682" layout="responsive"></amp-img>
-        <amp-img src="/img/product1_alt2_1024x682.jpg" width="1024" height="682" layout="responsive"></amp-img>
+   </div>
+   <!-- #### Image Gallery -->
```

Comment:
idea: we don't need to show all three carousels in the source code view. We could move the documentation before the first carousel.